### PR TITLE
Openvidu browser now identifies iPad Pro as a supported device

### PR DESCRIPTION
**Pull request by rgustavsson**
Openvidu browser didn't treat iPad Pro as a supported device.

The problem was that iPad Pro didn't fullfill the `platform.product === 'iPad'` statement (`product` from Platform.js was `null`).

We already had a custom check for iPad (incl. Pro) that was working so I basically used this instead of Platform.js when setting platform['isIonicIos']

This fix solves issue 529.

### CODE REVIEW

**Comment by CSantosM** [comment]
Thank you for the PR @rgustavsson ,

Have you tested with different iPad models using ionic and using OpenVidu through Safari browser and everything work as expected?

File: openvidu-browser/src/OpenVidu/OpenVidu.ts [hunk 1]
```diff
@@ -43,7 +43,11 @@ import RpcBuilder = require('../OpenViduInternal/KurentoUtils/kurento-jsonrpc');
  */
 import platform = require('platform');
 
-platform['isIonicIos'] = (platform.product === 'iPhone' || platform.product === 'iPad') && platform.ua!!.indexOf('Safari') === -1;
+const isTouchable = 'ontouchend' in document;
```
Comment (scorp-dmitriy) [inline openvidu-browser/src/OpenVidu/OpenVidu.ts:46]: Concerned about having this code at init of the file. Are you sure platform will be available? Maybe better to keep it inside the function as it was?

Comment (rgustavsson) [reply openvidu-browser/src/OpenVidu/OpenVidu.ts:46]: Yes from my debugging sessions I could see that platform was initialised, but as mentioned, `platform.product` isn't set to `iPad` for iPad Pro's. The check for iPad in `platform['isIonicIos']` was never in a function before. We've had "our own" function to check for iPad's in other places though.

**Comment by rgustavsson** [comment]
@CSantosM Thanks for the feedback. I'll update this PR as soon as I can with test information.
I only have one iPad and it's a Pro. If I publish my app so you can access it, would you be able to try it with one of your "normal" ipads om Safari? I am guessing it shouldn't take more than a couple of minutes. Else I am not sure how to continue. I'll look into the opportunity of buying one and in that case, which one would you recommend?

**Comment by rgustavsson** [comment]
@CSantosM 
Apologies for the delay. We are in UAT at the moment, expecting to test this thoroughly and release in a day or two. I'll share test results as soon as we're done.

**Comment by anupzone** [comment]
checked in ipad app which is using safari 14, issue still there. I found in code platform.js for detecting browsers and apps which is supporting till safari 13 and that's why giving browser null in return. posted issue here also:
[https://openvidu.discourse.group/t/ipad-pro-still-giving-same-error-in-2-15-0-version/2140](url)

**Comment by CSantosM** [comment]
Fixed on master (#550).
It'll be included in the next release (2.16.0)

**Pull request closed without merging**